platform/ios/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed an issue causing a composite function’s highest zoom level stop to be misinterpreted. ([#8613](https://github.com/mapbox/mapbox-gl-native/pull/8613), [#8790](https://github.com/mapbox/mapbox-gl-native/pull/8790))
 * Fixed an issue where re-adding a layer that had been previously removed from a style would reset its paint properties. Moved initializers for `MGLTileSource`, `MGLStyleLayer`, and `MGLForegroundStyleLayer` to their concrete subclasses; because these classes were already intended for initialization only via concrete subclasses, this should have no developer impact. ([#8626](https://github.com/mapbox/mapbox-gl-native/pull/8626))
 * Feature querying results now account for any changes to a feature’s size caused by a source or composite style function. ([#8665](https://github.com/mapbox/mapbox-gl-native/pull/8665))
+* Added support for 3D extrusion's light via the `MGLLight` class and the `light` property in style JSON. ([#9043](https://github.com/mapbox/mapbox-gl-native/pull/9043))
```

Comment:
There's already a blurb above about adding 3D extrusion support to the release. This is one part of that feature, not a separate feature. Let's remove this entry and put it in the draft release notes for beta 1. (But feel free to add a link to this PR to the end of the existing blurb.)